ale_linters/ocaml/merlin.vim

```diff
@@ -0,0 +1,20 @@
+" Author: Andrey Popp -- @andreypopp
+" Description: Report errors in OCaml code with Merlin
+
+if !exists('g:merlin')
+  echo 'Error: merlin vim bindings are required for Ale merlin linter to work'
```

Comment:
That error might be a bit too annoying, as it will keep appearing until someone installs the other plugin, which they might not want. I'd put this in the documentation somewhere. Maybe something like `(see |ale-merlin-integration|)` in the Vim documentation and something similar in the README file, maybe just tell people to type `:help ale-merlin-integration` if you want to avoid writing the same text twice.